### ISSUE TITLE
"waiting on XR startup" warning spammed in Editor

### ISSUE THREAD

**Issue by SimonStHilaire** [microsoft/MixedReality-WorldLockingTools-Unity#221]
As soon as the game is launched, the console is flooded with this message, even when the game is stopped.

**Comment by fast-slow-still** [comment]
I don't have any inside knowledge on this, but I have seen it myself. My *theory* is that a system service has terminated, preventing XR from starting up correctly. Your experience suggests a watchdog process eventually restarts the service. I have found that restarting the HoloLens (or phone) also fixes the problem.

**Comment by SimonStHilaire** [comment]
It happens regularly and it is flooding the console.

**Comment by fast-slow-still** [comment]
Can you give some context? Device, versions of Unity, WLT, etc. See [this doc](https://microsoft.github.io/MixedReality-WorldLockingTools-Unity/DocGen/Documentation/HowTos/Contributing.html#reporting-a-bug) for guidance on how to make a bug report most actionable.

Also, please include any steps leading up to this, anything that will help me reproduce the bug will make it possible to fix the bug. Thanks!

**Comment by zantiu** [comment]
Isn't this the same as https://github.com/microsoft/MixedReality-WorldLockingTools-Unity/issues/223 ?
With a proposed solution here: https://github.com/microsoft/MixedReality-WorldLockingTools-Unity/issues/225

**Comment by SimonStHilaire** [comment]
Yes it is. And sorry for the low quality bug report. I will work harder next time I promise :)

Duplicate of [#223]
